packages/react/src/components/Notification/Notification.tsx

```diff
@@ -567,17 +804,117 @@ InlineNotification.propTypes = {
    */
   title: PropTypes.string,
 };
-
 InlineNotification.defaultProps = {
   kind: 'error',
   role: 'status',
   onCloseButtonClick: () => {},
   hideCloseButton: false,
 };
 
+/**
+ * ActionableNotification
+ * ======================
+ */
+
+export interface ActionableNotificationProps {
```

Comment:
should this extend Div attributes? extra props can be passed in through `..rest`